style.css

```diff
@@ -22,9 +23,13 @@ img {
 
 .header {
   order: 1;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
```

Comment:
- [ ] Good job with your design kindly make your design fills the webpage and looks as close as possible to the Figma design. Currently your desktop site isn't responsive screen size more than `768px`kindly fix this 🙏 

Your Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/186783334-c7172226-fe04-4112-9d46-2eddb8757354.png)  |  ![image](https://user-images.githubusercontent.com/74814780/186783303-28ea978b-760c-48f3-9fef-b5330ebff22c.png)

Reply:
Done!